dev/Vagrantfile

```diff
@@ -7,7 +7,7 @@ CS_VM_ADDRESS="192.168.33.100"
 DB_VM_ADDRESS="192.168.33.150"
 DB_SUPERUSER="bofh"
 DB_SUPERPASS="i1uvd3v0ps"
-
+CS_VM_2_ADDRESS="192.168.33.101"
```

Comment:
By th way, more ugliness ^ :D Simplest way to make dvm work with a second FE, but there are some things we can do much better for dvm if we want to support various topos.